test/support/schema.exs

```diff
@@ -131,6 +131,7 @@ defmodule TestExAdmin.Simple do
   schema "simples" do
     field :name, :string
     field :description, :string
+    field :exists, :boolean, virtual: true
```

Comment:
I'm not sure if this matters, but for the sake of completeness, in my use case, this would be `:exists?`